extensions/http-curl/tests/HTTPHandlers.h

```diff
@@ -206,25 +207,36 @@ class FlowFileResponder : public CivetHandler {
       minifi::io::CivetStream civet_stream(conn);
       minifi::io::CRCStream < minifi::io::CivetStream > stream(&civet_stream);
       uint32_t num_attributes;
+      int read;
       uint64_t total_size = 0;
-      total_size += stream.read(num_attributes);
+      read = stream.read(num_attributes);
+      if(!isServerRunning())return false;
```

Comment:
Why?

I would expect not even trying to read the stream when the server is not running.

Reply:
apparently they do not segfault, but simply indicate error in their return value [mg_read](https://github.com/civetweb/civetweb/blob/master/docs/api/mg_read.md)